src/Illuminate/Queue/Queue.php

```diff
@@ -288,4 +320,18 @@ public function setContainer(Container $container)
     {
         $this->container = $container;
     }
+
+    /**
+     * Raise the job queued event.
+     *
+     * @param string|object $job
```

Comment:
cs

Reply:
Not sure about your camment, I put `string|object` because both `puch()` and `later()` use this signature for `$job`

Reply:
I mean you need 2 spaces before and after the types in param docs, just like every other param doc in the framework.